modules/installing-ocp-agent-gather-log.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/installing-with-agent-based-installer.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-gather-log_{context}"]
+= Gathering log data from a failed Agent-based installation
+
+Use the following procedure to gather log data about a failed Agent-based installation to provide for a support case.
+
+.Procedure
+
+. Run the following command and collect its output:
++
+[source,terminal]
+----
+$ ./openshift-install --dir <install_directory> agent wait-for bootstrap-complete --log-level=debug
+----
+
+. If the output from the previous command indicates a failure, or if the bootstrap is not progressing, run the following command on node 0 and collect its output:
```

Comment:
Do you think its worth adding an example output of what a failed or successful collection of an output? Or even maybe a brief description of it? I can see doing that might over crowd the procedure cuz I'm not sure what a failed or successful collection looks like, but I was curious about your thoughts as well.

Reply:
It might be worth it, I'll ask an SME to see if there's a short and easy example to mention. It's tough because this is sort of an open-ended procedure, where likely any number of things can indicate a failure, and maybe it's buried in a lot of output.

But if there's an easy example to mention from the SME, I can try and include it here.

Reply:
Yeah I've had that issue before with huge potential output. I've sometimes tried to put breifish (1-2 sentences) descriptions if the example output is too crazy. Even if there is something to show for a successful build can also be helpful bc users can have some kind of reference to go off of. Users can also probably tell if the command failed by comparing their output to the successful example, or vise versa. I hope that helps!

Reply:
I was able to get an example of an error message and added it to the previous step, where you run the command that would give you the message in the first place.